src/routes/articles.js

```diff
@@ -107,6 +107,40 @@ const getArticle = async (req, res, next) => {
 	}
 }
 
+// UPDATE ARTICLES ENDPOINT
+
+const updateArticle = async (req, res, next) => {
+	const { id } = req.params // WHERE
+	const { title, article, image, published } = req.body // SET
+
+	try {
+		const updatedArticle = await postService.updatePost({
+			title,
+			content : article,
+			image,
+			published,
+			id
+		})
+		if (!updatedArticle) {
+			return res.status(404).json({
```

Comment:
kindly move the error into the service as we did for the other one

Reply:
yes please